django_ssl_auth/base.py

```diff
@@ -101,7 +101,7 @@ def process_request(self, request):
                 return
             logger.debug("REST API call, not logging user in")
             request.user = user
-        elif request.path_info == settings.LOGIN_URL:
+        elif request.path_info == settings.LOGIN_URL or request.path_info == settings.SSLCLIENT_LOGIN_URL:
```

Comment:
Currently, this will fail if `SSLCLIENT_LOGIN_URL` is not set. This means people can not simply upgrade.

I would like to see
1. This should continue to work as before when the setting is not present
2. Tests to make sure this new setting works as expected

Reply:
Made the change you requested and added a unit test.  The travis CI stuff failed, but not because of my changes.